pokemongo_bot/cell_workers/update_live_stats.py

```diff
@@ -262,7 +262,8 @@ def _get_player_stats(self):
         :return: The player stats object.
         :rtype: dict
         """
-        inventory_items = self.bot.get_inventory() \
+        # TODO : find a better solution than calling the api
+        inventory_items = self.bot.api.get_inventory() \
```

Comment:
@BriceSD I don't know if we can avoid calling the API for the player stats. Are they stored and correctly updated internally ? In which case we could use the internally stored info instead of calling the API.
If no such thing is available, then I guess the same kind of refactoring is necessary to get the player data once and update it internally.